Descent3/lnxmain.cpp

```diff
@@ -322,6 +322,11 @@ int d3SDLEventFilter(const SDL_Event *event) {
     return (sdlMouseButtonUpFilter(event));
   case SDL_MOUSEBUTTONDOWN:
     return (sdlMouseButtonDownFilter(event));
+  case SDL_QUIT:
+    SDL_Quit();
+    _exit(0);
+    break;
+  default: break;
```

Comment:
I would not include a default case instead of including a default case that doesn’t do anything.

Reply:
It _does_ do something: it prevent compiler warnings since we have a `switch` on an enum when we (intentionally) don't handle every value in that enum.